frontend/src/global_styles/content/_projects_list.sass

```diff
@@ -134,3 +137,14 @@ body.controller-projects.action-index
     padding-left: 1em
   li
     list-style-type: none
+
+.project-list-container
+  display: flex
+  flex-direction: column
+  max-height: calc(100vh - #{$toolbar-height})
+
+.project-list
+  display: flex
+  flex-grow: 1
+  overflow: scroll
```

Comment:
Please use `overflow: auto` instead, as we don't want a scrollbar when the table is small